The lock file does not exist. Locking.
Creating virtualenv whisper-segmentation in /home/ryu/Documents/Code/whisper-segmentation/.venv
Updating dependencies
Resolving dependencies...

  CalledProcessError

  Command '['git', '--git-dir', '/tmp/pypoetry-git-whisperehkulg8c/.git', '--work-tree', '/tmp/pypoetry-git-whisperehkulg8c', 'checkout', 'master']' returned non-zero exit status 1.

  at ~/.local/pipx/venvs/poetry/lib/python3.10/site-packages/poetry/utils/_compat.py:217 in run
      213│                 process.wait()
      214│                 raise
      215│             retcode = process.poll()
      216│             if check and retcode:
    → 217│                 raise CalledProcessError(
      218│                     retcode, process.args, output=stdout, stderr=stderr
      219│                 )
      220│         finally:
      221│             # None because our context manager __exit__ does not use them.
